release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -85,11 +85,133 @@ The {product-title} installation program for Google Cloud Platform (GCP) now cre
+
+[discrete]
+===== Restrictions
+
+Note the following restrictions for {product-title} on IBM Z and LinuxONE:
+
+* {product-title} for IBM Z does not include the following Technology Preview features:
+** Precision Time Protocol (PTP) hardware
+
+* The following {product-title} features are unsupported:
+** Automatic repair of damaged machines with machine health checking
+** CodeReady Containers (CRC)
+** Controlling overcommit and managing container density on nodes
+** CSI volume cloning
+** CSI volume snapshots
+** FIPS cryptography
+** Helm command-line interface (CLI) tool
+** Multus CNI plug-in
+** NVMe
+** OpenShift Metering
+** OpenShift Virtualization
+** Tang mode disk encryption during {product-title} deployment
+
+* Worker nodes must run {op-system-first}.
+* Persistent shared storage must be provisioned by using either NFS or other supported storage protocols
+* Persistent non-shared storage must be provisioned using local storage, like iSCSI, FC, or using LSO with DASD, FCP, or EDEV/FBA.
+
+[id="ocp-4-8-ibm-power"]
+==== IBM Power Systems
+
+With this release, IBM Power Systems are now compatible with {product-title} {product-version}.
+
+* xref:../installing/installing_ibm_power/installing-ibm-power.adoc#installing-ibm-power_installing-ibm-power[Installing a cluster on IBM Power Systems]
+* xref:../installing/installing_ibm_power/installing-restricted-networks-ibm-power.adoc#installing-restricted-networks-ibm-power_installing-restricted-networks-ibm-power[Installing a cluster on IBM Power Systems in a restricted network] for installation instructions.
+
+[discrete]
+===== Notable Enhancements
+
+The following new features are supported on IBM Power Systems with {product-title} {product-version}:
+
+* Encrypting data stored in etcd
+* 3-node cluster support
+* Multus SR-IOV
+
+[discrete]
+===== Supported Features
+
+The following features are also supported on IBM Power Systems:
+
+* Currently, five Operators are supported:
+** Cluster-Logging-Operator
+** Cluster-NFD-Operator
+** Elastic Search-Operator
+** Local Storage Operator
+** SR-IOV Network Operator
+
+* Multipathing
+* Persistent storage using iSCSI
+* Persistent storage using local volumes (Local Storage Operator)
+* Persistent storage using hostPath
+* Persistent storage using Fibre Channel
+* Persistent storage using Raw Block
+* OVN-Kubernetes with an initial installation of {product-title} 4.8
+* 4K Disk Support
+
+[discrete]
+===== Restrictions
+
+Note the following restrictions for {product-title} on IBM Power Systems:
+
+* {product-title} for IBM Power Systems does not include the following Technology Preview features:
+** Precision Time Protocol (PTP) hardware
+
+** Automatic repair of damaged machines with machine health checking
+** CodeReady Containers (CRC)
+** Controlling overcommit and managing container density on nodes
+** FIPS cryptography
+** Helm command-line interface (CLI) tool
+** NVMe
```

Comment:
NVMe is supported on Power.

Reply:
I'll be adding it to supported feature list